README.md

```diff
@@ -63,7 +63,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://htmlpreview.github.io/?https://github.com/MasterLeiva/Portfolio/blob/master/index.html)
+- [Live Demo Link](https://htmlpreview.github.io/?https://github.com/MasterLeiva/DavidLeiva-Portfolio/blob/navigation-bar/index.html)
```

Comment:
- [ OPTIONAL ] Kindly remove the `server` section under tech stark as your project currently doesn't require to use server. this will make your readme look more professional.
Am requesting this change as optional because it was missed by the previous reviewer but I `highly` recommend making this change.
- [ REQUIRED ] Kindly provide a valid link to your issue page. currently, it is broken and doesn't work as expected. I would advise making this change before asking for another review. this is the only issue stopping you from getting approved, make this change and the next review will approve your request 💯